### DISCUSSION TITLE
about pointrend training

### DISCUSSION THREAD

**Discussion by Furkan4dSight** [facebookresearch/detectron2#2857] [Q&A]
Hey, 
I registered the custom dataset(person class) with person images to train the pointrend. The results became very bad. For instance, model cannot draw any bbox and masks. Could you tell me why the results are too bad. config file like below,

cfg = get_cfg()
    add_pointrend_config(cfg)
    #cfg.merge_from_file(args.config_file)
    cfg.merge_from_list(args.opts)
    cfg.DATASETS.TRAIN = ("person_data",)
    #cfg.merge_from_list(args.opts)
    cfg.DATALOADER.NUM_WORKERS = 2
    cfg.MODEL.WEIGHTS = "detectron2://PointRend/InstanceSegmentation/pointrend_rcnn_R_50_FPN_3x_coco/164955410/model_final_3c3198.pkl"
    cfg.SOLVER.IMS_PER_BATCH = 4
    cfg.SOLVER.CHECKPOINT_PERIOD = 500000
    #cfg.SOLVER.BASE_LR = 0.0002
    cfg.SOLVER.MAX_ITER = 1000000
    cfg.SOLVER.STEPS = (499999,599999,699999,799999,899999,999999) 
    cfg.MODEL.ROI_HEADS.BATCH_SIZE_PER_IMAGE = 512 
    cfg.MODEL.POINT_HEAD.NUM_CLASSES = 1
    cfg.MODEL.ROI_HEADS.NUM_CLASSES = 1
    cfg.freeze()
    default_setup(cfg, args)

Thank you.
